lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -3235,7 +3235,8 @@ class VJPEmitter final
     // This instruction is active. Determine the appropriate differentiation
     // strategy, and use it.
     auto *structDecl = sei->getStructDecl();
-    if (structDecl->getAttrs().hasAttribute<FieldwiseDifferentiableAttr>()) {
+    if (structDecl->getEffectiveAccess() <= AccessLevel::Internal
```

Comment:
Minor: place operators on the preceding line.
```suggestion
    if (structDecl->getEffectiveAccess() <= AccessLevel::Internal ||
```